examples/quickrefl/core/editors/importdatawidgets/famousloader/importdatastructure.cpp

```diff
@@ -0,0 +1,270 @@
+// ************************************************************************** //
+//
+//  Model-view-view-model framework for large GUI applications
+//
+//! @license   GNU General Public License v3 or higher (see COPYING)
+//! @authors   see AUTHORS
+//
+// ************************************************************************** //
+
+#include "importdatastructure.h"
+
+#include <cmath>
+
+namespace DataImport
```

Comment:
It would be nice to do something with `DataImport::DataStructure` to avoid too much `Data`. Please see nice summary of the naming [here](https://dzone.com/articles/naming-conventions-from-uncle-bobs-clean-code-phil).